src/tls13.c

```diff
@@ -12437,7 +12437,10 @@ int wolfSSL_read_early_data(WOLFSSL* ssl, void* data, int sz, int* outSz)
     if (ssl->options.handShakeState == NULL_STATE) {
         if (ssl->error != WC_PENDING_E)
             ssl->earlyData = expecting_early_data;
-        ret = wolfSSL_accept_TLSv13(ssl);
+        /* TODO: discuss if this is a good idea and will not break things */
```

Comment:
This needs fixed before we can merge. The TODO and also the use of `fprintf(stderr` due to non-portability.